contracts/Overtime/LiveTradingProcessor.sol

```diff
@@ -82,6 +82,7 @@ contract LiveTradingProcessor is ChainlinkClient, Ownable, Pausable {
         address _referrer,
         address _collateral
     ) external whenNotPaused {
+        //TODO: consider how to prevent someone spamming this method
         require(
```

Comment:
We can add a timeout per user. The timeout may increase if the user repeats the request in a shorter timeframe

Reply:
They can just sybil using different wallets then. I think its fine because the CL node will not send a response if the data is not valid, but wanted to leave the TODO there as a reminder to keep it in mind once we start end-to-end testing.